addons/kubernetes-dashboard/addon.yaml

```diff
@@ -51,3 +51,8 @@ spec:
     selector:
       k8s-addon: kubernetes-dashboard.addons.k8s.io
     manifest: v1.10.0.yaml
```

Comment:
I wonder if it makes sense to remove the old version? Thoughts?

Reply:
I agree, i think it's more more safe with the CVE
Will update my PR